pkg/cloudevents/transport/pubsub/transport.go

```diff
@@ -42,6 +43,9 @@ type Transport struct {
 
 	// PubSub
 
+	// ReceiveSettings is used to configure Pubsub pull subscription.
+	ReceiveSettings *pubsub.ReceiveSettings
```

Comment:
I think you still need a function that returns an Option to set this up. Similar to how the other fields are set?

Reply:
Didn't realize there is an option. Done.